### PR TITLE
LREC 2020 fixes

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

**Comment by mjpost** [comment]
Good catch.

After approval, let's leave this open a day or two to catch any other errors that trickle in.

**Comment by mjpost** [comment]
Rebranding this "LREC fixes" so we can gather and process them all at once.

**Comment by nschneid** [comment]
Some other missing diacritics in author names: (throughout, e.g. https://www.aclweb.org/anthology/2020.lrec-1.407/)

- Jan Hajič
- Inguna Skadiņa
- José Manuel Gómez-Pérez

**Comment by mjpost** [comment]
I think this is ready, if someone has time to review.

File: data/xml/2020.rail.xml [hunk 1]
```diff
@@ -18,11 +18,11 @@
       <url hash="22832a2e">2020.rail-1.0</url>
     </frontmatter>
     <paper id="1">
-      <title>Endangered <fixed-case>A</fixed-case>frican Languages Featured in a Digital Collection: The Case of the Khomani San, Hugh Brody Collection</title>
+      <title>Endangered <fixed-case>A</fixed-case>frican Languages Featured in a Digital Collection: The Case of the ǂKhomani San, Hugh Brody Collection</title>
```
Comment (nschneid) [inline data/xml/2020.rail.xml:21]: might as well make it:

```
ǂ<fixed-case>K</fixed-case>homani <fixed-case>S</fixed-case>an, <fixed-case>H</fixed-case>ugh <fixed-case>B</fixed-case>rody <fixed-case>C</fixed-case>ollection
```

**Pull request merged**